angular-client/templates/login.html

```diff
@@ -1,11 +1,55 @@
-	<!-- <div ng-view></div> -->
-	
-<h1>Login</h1>
-<div class="col-xs-3" ng-controller='login'>
-	<h3>Email</h3>
-<input name="text" class="form-control" ng-model="nameLog">
-	<h3>Password</h3>
-<input type="password" class="form-control" ng-model="passwordLog">
-<button class="btn btn-primary" ng-click="login()">LogIn</button>
+
+<div ng-controller='login'>
+
+	<style type="text/css">
+  
+body{
+  background-image: url(http://www.joburgchiropractor.co.za/images/background.jpg);
+}
+</style>
+<div class="container" style="margin-top: 5%;">
+  <div class="row">
+    <div class="col-sm-4"> </div>
+<div class="col-md-4">
```

Comment:
Spacing for this whole file needs to be fixed. When you place an element inside of another element, indent the new element further right.

<div class="div1">
    <div class="div2">
        <div class="div3"></div>
    </div>
</div>